src/components/Modal.js

```diff
@@ -27,11 +28,25 @@ const styles = {
 
 const Modal = ({ onDismiss, title, titleExtras, children, width = 450, showCancel = true, cancelText = 'Cancel', showX, showButtons = true, okButton, ...props }) => {
   const titleId = Utils.useUniqueId()
+  const modalNode = useRef()
+  const previouslyFocusedNode = useRef()
   // react-modal applies aria-hidden to the app root *and* takes care of limiting what can be tab-focused - see appLoader.js
   return h(RModal, {
```

Comment:
Since we're handling focus ourselves now, can we turn off the Modal's default focus handling to simplify things?